### PR TITLE
Made changes to .travis.yml and database.yml files

### PR DESCRIPTION
**Pull request by audreyshub**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+language: ruby
+sudo: false
+script: "bundle exec rake"
+
+before_script:
+- psql -c 'create database travic_ci_test;' -U postgres
```
Comment (kenmazaika) [inline .travis.yml]: looks like a typo here.  travic not travis :)

Comment (audreyshub) [inline .travis.yml]: Whoopsies! Tks :)

**Comment by kenmazaika** [comment]
TravisCI is passing.  This is how you should feel, Audrey!

boo yah

**Comment by audreyshub** [comment]
LOL =P

**Comment by bfhcodehacker** [comment]
I'm gonna merge this so I can start fresh with the full build when adding the models.

**Pull request merged**